Style.css

```diff
@@ -240,7 +240,6 @@ header nav div .menu-bar {
   padding-bottom: 12px;
 }
 
-
```

Comment:
#### I highlighted this piece of code because the relevant piece of code does not appear under the files changed section of this pull request.

- [ ] The gap between the last card and the _**About Myself**_ text is too wide. Kindly reduce it so that your design looks identical to that of the Figma template you chose. 👍 

### Your Design

![Screenshot 2023-01-12 200654](https://user-images.githubusercontent.com/77106962/212158554-2076a32a-d993-4b29-9d65-211835719a0e.png)

### Figma template

![image](https://user-images.githubusercontent.com/77106962/212158765-d7a547b6-2017-4266-bd7e-8dd03495b119.png)